### PR TITLE
bleachbit_console: add page

### PR DESCRIPTION
**Pull request by fnfontana**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 4.4.2

### CODE REVIEW

**Comment by fnfontana** [comment]
Add page for bleachbit_console command, also known as Bleachbit CLI, for Windows.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, Welcome to tldr. I have some suggestions for this page.

File: pages/windows/bleachbit_console.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# bleachbit_console
+
+> The Bleachbit command-line interface (CLI) allows users to automate the process of cleaning junk files on their system.
+> It can be run with various command-line arguments to perform different clean-up operations.
+> More information: <https://docs.bleachbit.org/doc/command-line-interface.html>.
+
+- Perform the clean-up operation and delete files:
+
+`bleachbit_console.exe --clean`
```
Comment (EmilySeville7cfg) [inline pages/windows/bleachbit_console.md:9]:
```suggestion
- Perform the clean-up operation and delete files:

`bleachbit_console --clean`
```

Optional fix: `.exe` can be removed.

File: pages/windows/bleachbit_console.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- Preview the files that will be deleted and other changes that will be made before actually performing the clean-up operation:
+
+`bleachbit_console.exe --preview`
```
Comment (EmilySeville7cfg) [inline pages/windows/bleachbit_console.md:13]:
```suggestion
`bleachbit_console --preview`
```

The same issue as above.

File: pages/windows/bleachbit_console.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+- Display system information:
+
+`bleachbit_console.exe --sysinfo`
```
Comment (EmilySeville7cfg) [inline pages/windows/bleachbit_console.md:17]:
```suggestion
`bleachbit_console --sysinfo`
```

The same issue as above.

File: pages/windows/bleachbit_console.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+- Start the graphical user interface (GUI) version of Bleachbit:
+
+`bleachbit_console.exe --gui`
```
Comment (EmilySeville7cfg) [inline pages/windows/bleachbit_console.md:21]:
```suggestion
`bleachbit_console --gui`
```

The same issue as above.

File: pages/windows/bleachbit_console.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+- Display version:
+
+`bleachbit_console.exe --version`
```
Comment (EmilySeville7cfg) [inline pages/windows/bleachbit_console.md:25]:
```suggestion
`bleachbit_console --version`
```

The same issue as above.

**Review by EmilySeville7cfg — APPROVED** [review]
No mandatory fixes found, approved.
Thanks for a contribution! ❤️

File: pages/windows/bleachbit_console.md (hunk 1 above)
Comment (kbdharun) [reply pages/windows/bleachbit_console.md:9]: Generally, for Bleachbit in Windows `.exe` extension is required, and it's their recommended method so I will mark these suggestions as resolved. For using without exe you need to add it is an environment variable.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**